lib/Runtime/Language/InterpreterStackFrame.h

```diff
@@ -4,8 +4,13 @@
 //-------------------------------------------------------------------------------------------------------
 #pragma once
 
+#ifdef _WIN32
```

Comment:
This should be _MSC_VER, since the intrinsic being ifdefed is a VC++ one, not a Win32 specific one